container/push.bzl

```diff
@@ -205,3 +215,26 @@ container_push = rule(
         "digest": "%{name}.digest",
     },
 )
+
+# Pushes a container image to a registry.
+def container_push(name, format, image, registry, repository, **kwargs):
+    _container_push(
+        name = name,
+        format = format,
+        image = image,
+        registry = registry,
+        repository = repository,
+        extension = select({
+            "@bazel_tools//src/conditions:host_windows": ".bat",
+            "//conditions:default": "",
+        }),
+        tag_tpl = select({
+            "@bazel_tools//src/conditions:host_windows": Label("//container:push-tag.bat.tpl"),
+            "//conditions:default": Label("//container:push-tag.sh.tpl"),
+        }),
+        windows_paths = select({
+            "@bazel_tools//src/conditions:host_windows": True,
+            "//conditions:default": False,
+        }),
```

Comment:
![image](https://user-images.githubusercontent.com/8533249/94815765-e9747c80-03c8-11eb-8af6-5a02b5ea8e3e.png)

Reply:
More like I have no idea how this bit works